frontend/src/app/__snapshots__/app.test.tsx.snap

```diff
@@ -219,6 +254,10 @@ exports[`App tests should render default App component 1`] = `
               }
             />
           </DataListItemRow>
+          <DataListItemRow
+            className=""
+            id="data-rows"
+          />
```

Comment:
Does this empty row serve a purpose?

Reply:
This is the same file created by the test.